skyline-vscode/react-ui/src/sections/EnergyConsumption.js

```diff
@@ -184,13 +212,18 @@ const EnergyConsumption = ({ energyData }) => {
                             />{" "}
                           </div>
                           <p>
-                            <strong>{conversions?.household} x</strong> homes'
-                            energy use for one year
+                            <strong>{conversions?.household[0]} x</strong> homes'
+                            energy use for one {conversions?.household[1]}
                           </p>
                         </Col>
                         <small>
-                          ref:
-                          https://www.epa.gov/energy/greenhouse-gas-equivalencies-calculator
+                          <a
+                            href="
+                          https://www.epa.gov/energy/greenhouse-gas-equivalencies-calculator"
+                            target="_blank"
+                          >
+                            https://www.epa.gov/energy/greenhouse-gas-equivalencies-calculator
```

Comment:
Modify it to a  line that say something like "Greenhouse gas equivalencies are based on calculations provided by [the EPA](https://www.epa.gov/energy/greenhouse-gas-equivalencies-calculator)"